lib/filename_generator.ex

```diff
@@ -1,28 +1,35 @@
-defmodule MassagerFilenameGenerator do
+defmodule Massager.FilenameGenerator do
   # For example, this will turn: "input_path/order_1.csv" into {"input_path/order_1.csv", "input_path/order_01.csv"}
   def generate_filename_pairs(%{
         input_path_prefix: input_path_prefix,
         output_path_prefix: output_path_prefix,
-        prefix: prefix,
-        suffix: suffix,
-        starting_number: starting_index,
+        filename_prefix: filename_prefix,
```

Comment:
I prefer more descriptive names... I find code easier to understand months later, although perhaps too wordy?